scripts/helper/docker-generate-certificates.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
Not coherent with the /usr/bin/env calls elsewhere.

Reply:
replaced with #!/usr/bin/env bash for now (see chat discussion)